nikola/plugin_categories.py

```diff
@@ -471,3 +472,260 @@ def import_file(self):
+
+    generate_atom_feeds_for_post_lists = False:
+        Whether to generate Atom feeds for post lists in case GENERATE_ATOM is set.
+
+    template_for_list_of_one_classification = "tagindex.tmpl":
+        The template to use for the post list for one classification.
+
+    template_for_classification_overview = "list.tmpl":
+        The template to use for the classification overview page.
+        Set to None to avoid generating overviews.
+
+    always_disable_rss = False:
+        Whether to always disable RSS feed generation
+
+    apply_to_posts = True:
+        Whether this classification applies to posts.
+
+    apply_to_pages = False:
+        Whether this classification applies to pages.
+
+    minimum_post_count_per_classification_in_overview = 1:
+        The minimum number of posts a classification must have to be listed in
+        the overview.
+
+    omit_empty_classifications = False:
+        Whether post lists resp. indexes should be created for empty
+        classifications.
+
+    also_create_classifications_from_other_languages = True:
+        Whether to include all classifications for all languages in every
+        language, or only the classifications for one language in its language's
+        pages.
+    """
+
+    name = "dummy_taxonomy"
+
+    # Adjust the following values in your plugin!
+    classification_name = "taxonomy"
+    metadata_name = "taxonomy"
+    overview_page_variable_name = "taxonomy"
+    more_than_one_classifications_per_post = False
+    has_hierarchy = False
+    include_posts_from_subhierarchies = False
+    include_posts_into_hierarchy_root = False
+    show_list_as_subcategories_list = False
+    show_list_as_index = False
+    generate_atom_feeds_for_post_lists = False
+    template_for_list_of_one_classification = "tagindex.tmpl"
+    template_for_classification_overview = "list.tmpl"
+    always_disable_rss = False
+    apply_to_posts = True
+    apply_to_pages = False
+    minimum_post_count_per_classification_in_overview = 1
+    omit_empty_classifications = False
+    also_create_classifications_from_other_languages = True
+
+    def is_enabled(self, lang=None):
+        """Return True if this taxonomy is enabled, or False otherwise.
+
+        If lang is None, this determins whether the classification is
+        made at all. If lang is not None, this determines whether the
+        overview page and the classification lists are created for this
+        language.
+        """
+        return True
+
+    def get_implicit_classifications(self, lang):
+        """Return a list of classification strings which should always appear in posts_per_classification."""
+        return []
+
+    def classify(self, post, lang):
+        """Classify the given post for the given language.
+
+        Must return a list or tuple of strings.
+        """
+        raise NotImplementedError()
+
+    def sort_posts(self, posts, classification, lang):
+        """Sort the given list of posts.
```

Comment:
This allows the plugin to sort the posts by some other predicate than timestamp (the default sorting). I'll improve documentation.